network changes

DaMandal0rian opened the pull request:
- devnet changes
- upgrade snapshot-sep-25
- minor fixes to some variables
- enable timekeeper on farmer node
- add evm domain bootstrap node
- remove variable in tfvars
- add bootstrap evm node output
- adjustments for evm bootstrap node
- remove datadog remanants

closes #182

DaMandal0rian committed:
network changes

- devnet changes
- upgrade snapshot-sep-25
- minor fixes to some variables
- enable timekeeper on farmer node
- add evm domain bootstrap node
- remove variable in tfvars
- add bootstrap evm node output
- adjustments for evm bootstrap node
- remove datadog remanants

vedhavyas commented on aws/devnet/main.tf:
```diff
@@ -9,7 +10,23 @@ module "devnet" {
+    docker-tag         = "snapshot-2023-sep-25"
+    reserved-only      = false
+    prune              = false
+    genesis-hash       = ""
```
if this is empty, then do we need to have this variable ?

DaMandal0rian replied:
I use this to set in the `.env` file and populate it when running the node

DaMandal0rian replied:
It's populated in the .env file after.

vedhavyas commented on aws/network-primitives/scripts/create_bootstrap_node_evm_compose_file.sh:
```diff
@@ -119,17 +123,19 @@ cat >> ~/subspace/docker-compose.yml << EOF
-  if [ "${current_node}" != "${i}" ]; then
-    addr=$(sed -nr "s/NODE_${i}_MULTI_ADDR=//p" ~/subspace/node_keys.txt)
+
+for (( i = 0; i < bootstrap_node_count; i++ )); do
```
This would also set self node key as reserved. I dont think that is what we want

DaMandal0rian replied:
This is the bootstrap Domain Operator node

vedhavyas reviewed (commented):
Make sense. Left some questions I'm not sure about

DaMandal0rian committed:
change farmer machine type

- more resources to run farmer timekeeper
- add cache-percentage to genesis farmer

DaMandal0rian committed:
remove merge conflict
